brokerapi/brokers/cloudsql/postgres-definition.go

```diff
@@ -29,129 +29,153 @@ const PostgresServiceId = "cbad6d78-a73c-432d-b8ff-b219a17a803a"
-				        "id": "b44f8294-b003-4a50-80c2-706858073f44",
-				        "name": "postgres-db-n1-standard-32",
-				        "free": false
-				    },
-				    {
-				        "service_properties": { "tier": "db-custom-64-245760", "max_disk_size": "10230" },
-				        "description": "PostgreSQL with 64 CPUs, 240 GB/RAM, 10230 GB/disk, supporting 4,000 connections.",
-				        "id": "d97326e0-5af2-4da5-b970-b4772d59cded",
-				        "name": "postgres-db-n1-standard-64",
-				        "free": false
-				    },
-				    {
-				        "service_properties": {"tier": "db-custom-2-13312", "max_disk_size": "10230" },
-				        "description": "PostgreSQL with 2 CPUs, 13 GB/RAM, 10230 GB/disk, supporting 4,000 connections.",
-				        "id": "c10f8691-02f5-44eb-989f-7217393012ca",
-				        "name": "postgres-db-n1-highmem-2",
-				        "free": false
-				    },
-				    {
-				        "service_properties": { "tier": "db-custom-4-26624", "max_disk_size": "10230" },
-				        "description": "PostgreSQL with 4 CPUs, 26 GB/RAM, 10230 GB/disk, supporting 4,000 connections.",
-				        "id": "610cc78d-d26a-41a9-90b7-547a44517f03",
-				        "name": "postgres-db-n1-highmem-4",
-				        "free": false
-				    },
-				    {
-				        "service_properties": { "tier": "db-custom-8-53248",  "max_disk_size": "10230" },
-				        "description": "PostgreSQL with 8 CPUs, 52 GB/RAM, 10230 GB/disk, supporting 4,000 connections.",
-				        "id": "2a351e8d-958d-4c4f-ae46-c984fec18740",
-				        "name": "postgres-db-n1-highmem-8",
-				        "free": false
-				    },
-				    {
-				        "service_properties": { "tier": "db-custom-16-106496", "max_disk_size": "10230" },
-				        "description": "PostgreSQL with 16 CPUs, 104 GB/RAM, 10230 GB/disk, supporting 4,000 connections.",
-				        "id": "51d3ca0c-9d21-447d-a395-3e0dc0659775",
-				        "name": "postgres-db-n1-highmem-16",
-				        "free": false
-				    },
-				    {
-				        "service_properties": { "tier": "db-custom-32-212992", "max_disk_size": "10230" },
-				        "description": "PostgreSQL with 32 CPUs, 208 GB/RAM, 10230 GB/disk, supporting 4,000 connections.",
-				        "id": "2e72b386-f7ce-4f0d-a149-9f9a851337d4",
-				        "name": "postgres-db-n1-highmem-32",
-				        "free": false
-				    },
-				    {
-				        "service_properties": { "tier": "db-custom-64-425984", "max_disk_size": "10230" },
-				        "description": "PostgreSQL with 64 CPUs, 416 GB/RAM, 10230 GB/disk, supporting 4,000 connections.",
-				        "id": "82602649-e4ac-4a2f-b80d-dacd745aed6a",
-				        "name": "postgres-db-n1-highmem-64",
-				        "free": false
-				    }
-				]
-    }`,
+		Id:               "cbad6d78-a73c-432d-b8ff-b219a17a803a",
+		Name:             "google-cloudsql-postgres",
+		Description:      "Google Cloud SQL is a fully-managed PostgreSQL database service.",
```

Comment:
@josephlewis42 is this correct? Doesn't seem to match the diff

Reply:
Nice catch, it would help if I `push`ed all the updates before I completed the PR. :-)